src/index.ts

```diff
@@ -276,6 +279,34 @@ class Monika extends Command {
           em.emit('SANITIZED_CONFIG', sanitizedProbe)
         }
 
+        // run TLS checker
+        if (config?.certificate && config.certificate?.domains.length > 0) {
+          config.certificate?.domains.forEach(async (domain) => {
```

Comment:
Since we have checked `config.certificate?.domains.length > 0` before, we can rest assure that config.certificate will not empty, thus we can remove `?` mark after `certificate`.

Reply:
Thank you, nice catch. I fixed it here [3ebf505](https://github.com/hyperjumptech/monika/pull/315/commits/3ebf505c1b098b7c70c7d335accdd82a689d6d63).